apps/storybook/src/stories/datacom-avatar.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { ComponentStoryFn, Meta } from '@storybook/react';
+import { DatacomAvataravatar } from '@datacom/endeavour-react';
+
+export default {
+  title: 'Avatar',
+  component: DatacomAvataravatar,
+  argTypes: {
+    firstName: {
+      name: 'First Name',
+      defaultValue: 'First Name',
+      description: 'Button text',
+      type: { name: 'string', required: true },
+    },
+    lastName: {
+      name: 'Last Name',
+      defaultValue: 'Button text',
+      description: 'Button text',
+      type: { name: 'string', required: true },
+    },
+    src: {
+      name: 'Image Source',
+      defaultValue: 'Button text',
+      description: 'Button text',
+      type: { name: 'string', required: true },
+    },
+    jobTitle: {
+      name: 'Job title',
+      defaultValue: 'Button text',
+      description: 'Button text',
+      type: { name: 'string', required: true },
+    },
+    companyName: {
+      name: 'Company Name',
+      defaultValue: 'Button text',
+      description: 'Button text',
+      type: { name: 'string', required: true },
+    },
+  },
+  args: {
+    firstName: 'First Name',
+    lastName: 'Last Name',
+    src: 'Src',
+    jobTitle: 'Job title',
+    companyName: 'Datacom',
+  },
+} as Meta<typeof DatacomAvataravatar>;
```

Comment:
As mentioned previously, should probably fix all these double avatar instances (both in the code and file names)